status-panels-lib/jsonnetfile.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "git": {
+          "remote": "https://github.com/grafana/grafonnet.git",
+          "subdir": "gen/grafonnet-latest"
```

Comment:
I suggest we stick to v10.0.0 schema as of now, instead of pulling main.

Reply:
Fixed